devices.js

```diff
@@ -11607,6 +11607,44 @@ const devices = [
             await reporting.onOff(endpoint);
         },
     },
+    {
+        zigbeeModel: ['ZG2819S-CCT'],
+        model: 'ZG2819S-CCT',
+        vendor: 'Sunricher',
+        description: 'Zigbee handheld remote CCT 4 channels',
+        fromZigbee: [fz.battery, fz.command_move_to_color, fz.command_move_to_color_temp, fz.command_move_hue,
+            fz.command_step, fz.command_recall, fz.command_on, fz.command_off,
+            fz.command_on, fz.command_off, fz.command_toggle, fz.command_stop, fz.command_move,
+            fz.command_color_loop_set, fz.command_ehanced_move_to_hue_and_saturation, fz.tint_scene],
```

Comment:
Is `tint_scene` actually needed here? Can you test if the scene commands also work without it?

Reply:
I'm not really sure, as I bound the remote to use it.
The scenes allows to "retrieve" current values and store the scene in the remote.

How could I test it?

Reply:
Remove `fz.fz.tint_scene` from this list and check if still all `action` are send correctly.

Reply:
It seems so.

I didn't see any error or unsupported message.
So safe to remove.